src/write.c

```diff
@@ -2199,5 +2215,15 @@ static void writeConfigBox(avifRWStream * s, avifCodecConfigurationBox * cfg, co
     // }
     avifRWStreamWriteU8(s, 0);
 
+    // According to section 2.2.1. of AV1 Image File Format specification v1.1.0,
+    // there is no need to write any OBU here.
+    // See https://aomediacodec.github.io/av1-avif/v1.1.0.html#av1-configuration-item-property.
+    // unsigned int (8) configOBUs[];
+}
+
+static void writeConfigBox(avifRWStream * s, const avifCodecConfigurationBox * cfg, const char * configPropName)
```

Comment:
The function split here is also uneven because almost all the code is in the writeCodecConfig() function.

Reply:
As written above, because this is partly to simplify https://github.com/AOMediaCodec/libavif/pull/1432. Also it matches the [section 2.3.3](https://aomediacodec.github.io/av1-isobmff/v1.2.0.html#av1codecconfigurationbox-syntax) which is also split into two functions.

Reply:
The two functions here also don't seem to match [section 2.3.3](https://aomediacodec.github.io/av1-isobmff/v1.2.0.html#av1codecconfigurationbox-syntax).